downloader/download.go

```diff
@@ -91,9 +87,8 @@ func (d *downloader) Download(files, dirs []string, readPath, writePath string,
 	for _, val := range dirs {
 		dirWPath := writePath + filepath.FromSlash(val)
 		dirRPath := readPath + val
-		dirPath := strings.TrimPrefix(strings.TrimSuffix(dirRPath, "/"), d.rootWorkingDirectoryServer)
 
-		if filter.CheckToFilter(dirPath, filterList) {
+		if filter.CheckToFilter(dirRPath, filterList) {
 			continue
```

Comment:
Duplicate: I might write an inline comment explaining that this continue statement skips the waitgroup add and goroutine creation. That's purely stylistic though, so I won't be offended if it isn't worth it to you.